app/assets/stylesheets/styles/components/guides/_guides.css.scss

```diff
@@ -95,87 +161,192 @@
     }
   }
 
-  .edit-link{
-    float: right;
-    margin-top: .9rem;
-  }
-
-  .overview{
-    margin: 1rem auto;
-  }
-
   .content {
     padding-bottom: 8rem;
   }
 
   dd.accordion-navigation > a{
-    border-top: rgb(148, 148, 148) 1px solid;
     background-color: transparent;
-    transition: .5s background-color;
+    border-bottom: rgb(148, 148, 148) 3px solid;
+    transition: 0.2s background-color;
 
     // angle icon
     i.fa {
-      margin-right: 8px;
       @include single-transition(transform, 0.2s, ease-out);
+      opacity: 0;
 
       &.open {
-        @include transform(rotate(90deg));
+        @include transform(rotate(180deg));
       }
     }
   }
 
+  // Show Arrow on Hover
+  .accordion > .accordion-navigation > a:hover i.fa{
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 5<br>Opening curly brace `{` should be preceded by one space